### ISSUE TITLE
when i make -j$(nproc)，there have an error

### ISSUE THREAD

**Issue by qgbdm** [popcornmix/omxplayer#823]
#

> I am sorry my English is poor and I hope you can understand my problem.The error is：linux/OMXAlsa.cpp:22:10: fatal error: IL/OMX_Core.h: ??????????????????
>    22 | #include <IL/OMX_Core.h>
>       |          ^~~~~~~~~~~~~~~
> compilation terminated.
> make: *** [Makefile:49??linux/OMXAlsa.o] ???? 1
> How should I solve it?
>

**Comment by popcornmix** [comment]
This repo is deprecated. See the README on front page.
It also has build instructions, including running ./prepare-native-raspbian.sh
Did you follow them?

**Comment by hesamira** [comment]
Hi,

 I have faced this error. please help me. I want to install omxplayer on raspberry pi 3b, 32 bit.
I saw the README on front page.
Thanks.

make: *** [Makefile:49: OMXSubtitleTagSami.o] error 1

**Comment by popcornmix** [comment]
omxplayer is not supported on bullseye.
It can be made to run with some effort (installing header files like `IL/OMX_Core.h`) on 32-bit bullseye, but it impossible on 64-bit.

**Comment by alreadyconfused** [comment]
@popcornmix Yeah alright i'll give it another go on the buster os. Thanks for the quick reply!
